src/interfaces/filing-interfaces/filing-interface.ts

```diff
@@ -13,110 +13,107 @@ import { DissolutionStatementTypes, DissolutionTypes } from '@/enums'
 
 /** Interface for incorporation filing data saved to the Legal API. */
 export interface IncorporationFilingIF {
-  filing: {
```

Comment:
If you "hide whitespace" then you'll see I simplified this by removing the "filing" object level.